### PR TITLE
change versions for feature/antares

### PR DESCRIPTION
**Pull request by brthor**
@eerhardt @livarcocc @piotrpMSFT

Let me know if you think of some other place this might need to change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,16 +40,16 @@ You can download .NET Core as either an installer (MSI, PKG) or a zip (zip, gzip
 
 |         |Version |.NET Core Installer|.NET Core SDK Installer|.NET Core Binaries|.NET Core SDK Binaries|
 |---------|:------:|:------:|:------:|:------:|:------:|
-|**Windows x64**|[![](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/Windows_x64_Release_version_badge.svg)](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/latest.version)|[Download](https://dotnetcli.blob.core.windows.net/dotnet/preview/Installers/Latest/dotnet-win-x64.latest.exe)|[Download](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-win-x64.latest.exe)|[Download](https://dotnetcli.blob.core.windows.net/dotnet/preview/Binaries/Latest/dotnet-win-x64.latest.zip)|[Download](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-win-x64.latest.zip)|
```
Comment (eerhardt) [inline README.md]: How about the `Build Status` too?  That way we can see if builds are green or not.

**Comment by brthor** [comment]
putting up jenkins here:
https://github.com/dotnet/dotnet-ci/pull/386

VSO is already going

File: README.md (hunk 1 above)
Comment (brthor) [inline README.md]: done.

**Comment by livarcocc** [comment]
:shipit:

**Comment by brthor** [comment]
test OSX x64 Release Build
test Ubuntu x64 Release Build

**Comment by brthor** [comment]
test OSX x64 Release Build please

**Pull request merged**